package/MDAnalysis/core/AtomGroup.py

```diff
@@ -3660,14 +3804,35 @@ def __init__(self, name, residues):
         self._cls = ResidueGroup
 
     @property
+    @deprecate(message="{}; use `segid` property instead".format(_SIXTEEN_DEPRECATION))
     def id(self):
         """Segment id (alias for :attr:`Segment.name`)"""
         return self.name
 
     @id.setter
+    @deprecate(message="{}; use `segid` property instead".format(_SIXTEEN_DEPRECATION))
     def id(self, x):
         self.name = x
 
+    @property
+    def segid(self):
+        """Segment id (alias for :attr:`Segment.name`)"""
+        return self.name
+
+    @id.setter
+    def segid(self, x):
+        self.name = x
+
+    @property
+    @deprecate(message="{}; use `segid` property instead".format(_SIXTEEN_DEPRECATION))
```

Comment:
I think git's messing up. I only added the ``segid`` property, which for the moment refers to segment ``name`` internally.